### PR TITLE
Fixed gap issues in `flower` icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?

- [ ] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [x] Other: Fixed gap issue

### CODE REVIEW

**Comment by jguddas** [comment]
I added my self to the contributors, since I had to put in some work to fix those issues.

File: icons/flower.svg [hunk 1]
```diff
@@ -9,10 +9,14 @@
...
-  <path d="M14.5 14.5 16 16" />
+  <path d="M12 16.5A4.5 4.5 0 1 1 7.5 12 4.5 4.5 0 1 1 12 7.5a4.5 4.5 0 1 1 4.5 4.5 4.5 4.5 0 1 1-4.5 4.5" />
+  <path d="M12 7.5V9" />
+  <path d="M7.5 12H9" />
+  <path d="M16.5 12H15" />
+  <path d="M12 16.5V15" />
+  <path d="m8 16 1.9-1.9" />
+  <path d="M14.1 9.9 16 8" />
+  <path d="m8 8 1.9 1.9" />
+  <path d="M14.1 14.1 16 16" />
```
Comment (karsa-mistmere) [inline icons/flower.svg:21]: The paths don't meet perfectly, and I think we could easily merge the outside shape as such:
```suggestion
  <path d="M7.5 12A4.5 4.5 0 1 1 12 7.5V9 7.5a4.5 4.5 0 1 1 4.5 4.5H15h1.5a4.5 4.5 0 1 1-4.5 4.5V15v1.5A4.5 4.5 0 1 1 7.5 12H9Z" />
  <path d="m8 8 1.88 1.88" />
  <path d="M14.12 9.88 16 8" />
  <path d="m8 16 1.88-1.88" />
  <path d="M14.12 14.12 16 16" />
```

Comment (jguddas) [reply icons/flower.svg:21]: I'm not a fan of overlapping lines, they can produce some black spots when antialiasing is enabled.

My optimizer removes them currently, but I know that we don't really have a guideline on them.

Comment (karsa-mistmere) [reply icons/flower.svg:21]: Weird, I don't think I've ever encountered this issue, can you provide an example?

I'm fine with keeping them separate, in that case, just the last four diagonal lines need to be fixed:
```suggestion
  <path d="M12 16.5A4.5 4.5 0 1 1 7.5 12 4.5 4.5 0 1 1 12 7.5a4.5 4.5 0 1 1 4.5 4.5 4.5 4.5 0 1 1-4.5 4.5" />
  <path d="M12 7.5V9" />
  <path d="M7.5 12H9" />
  <path d="M16.5 12H15" />
  <path d="M12 16.5V15" />
  <path d="m8 8 1.88 1.88" />
  <path d="M14.12 9.88 16 8" />
  <path d="m8 16 1.88-1.88" />
  <path d="M14.12 14.12 16 16" />
```

**Review by karsa-mistmere — APPROVED** [review]

**Pull request merged**